tests/test_radius_server_group.rb

```diff
@@ -0,0 +1,101 @@
+#
+# Minitest for RadiusServerGroup class
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require File.expand_path('../ciscotest', __FILE__)
+require File.expand_path('../../lib/cisco_node_utils/radius_server', __FILE__)
+
+# TestRadiusServerGroup - Minitest for RadiusServerGroup node utility.
+class TestRadiusServerGroup < CiscoTestCase
+  def setup
+    # setup runs at the beginning of each test
+    super
+    config('radius-server host 8.8.8.8',
+           'radius-server host 9.9.9.9',
+           'radius-server host 10.10.10.10',
+           'radius-server host 11.11.11.11',
+           'radius-server host 12.12.12.12',
+           'radius-server host 13.13.13.13')
+    no_radiusserver
+  end
+
+  def teardown
+    # teardown runs at the end of each test
+    no_radiusserver
+    super
+  end
+
+  def no_radiusserver
+    # Turn the feature off for a clean test.
+    config('no aaa group server radius red',
+           'no aaa group server radius blue')
+  end
+
+  # TESTS
+
+  def test_radiusserver_create_destroy_single
```

Comment:
I know you modeled this on other test files but we've recently started to clean up these long test method names. The provider names are redundant so we're removing them:

`/test_radiusserver_create_destroy_single/test_create_destroy_single/`